src/psyclone/domain/gocean/nodes/gocean_extract_node.py

```diff
@@ -235,12 +305,15 @@ def create_driver(self, input_list, output_list):
         for kernel in self.psy_data_body.walk(CodedKern):
             kernel.clear_cached_data()
 
-        # Recreate the instrumented region:
+        # Recreate the instrumented region. Due to the changes in the
+        # config files, fields and properties will now become local
+        # plain arrays and variables:
```

Comment:
Nice!

Reply:
Well, this is the part I am not really proud of - it feels too much like a hack tbh :( But ... it is kind of elegant to avoid coding too much.